heroku-longpolling.ts

```diff
@@ -0,0 +1,53 @@
+/* 
+1. Create file named bot.ts
+---------------------------------------------- */
+import { Bot } from 'grammy';
+
+const bot = new Bot(''); // <-- put your bot token between the ''
```

Comment:
Don't commit the token.
```suggestion
if (process.env.BOT_TOKEN == null) throw new TypeError('BOT_TOKEN missing.');
const bot = new Bot(process.env.BOT_TOKEN);
```

Reply:
done